misc/scripts/install-local.sh

```diff
@@ -154,6 +154,22 @@ function compare_remote_version() (
     fi
 )
 
+function get_incompatible_releases() {
+    local distro_name="$(lsb_release -si 2> /dev/null | tr '[:upper:]' '[:lower:]')"
```

Comment:
This doesn't take into account what distro your system is based on. So if `incompatible=("ubuntu:jammy")`, it would install on Pop_os jammy

Reply:
Accepting `"*:jammy"` as key would also work

Reply:
Perhaps it be more helpful if the (release) codename and the OS ID (distribution)   were given as completely separate fields, rather than as a single compound field.

Reply:
> This doesn't take into account what distro your system is based on. So if `incompatible=("ubuntu:jammy")`, it would install on Pop_os jammy

Any idea how to fix this?

Reply:
I think the suggestion was a way to specify a particular release codename, but still match all distributions.

Reply:
You can now do something like `incompatible=("*:jammy")`